core/injector/application_injector.cpp

```diff
@@ -465,7 +465,9 @@ namespace {
     auto log = log::createLogger("injector", "kagome");
 
     if (app_config.nodeKey()) {
-      log->info("Will use LibP2P keypair from config or args");
+      log->info(
+          "Will use LibP2P keypair from config or args (loading from node-key "
+          "flag)");
```

Comment:
Flag means option what has only true/false value
```suggestion
          "Will use LibP2P keypair from config or 'node-key' CLI arg");
```